CHANGES/5217.bugfix

```diff
@@ -0,0 +1 @@
+Updates serializers and tests to make plugin compatible with pulpcore-plugin 0.1.0rc4.
```

Comment:
I'm not sure what was incompatible with pulpcore, except for tests.
I didn't notice any changes in serializer (maybe black changes make it hard to notice).